### PR TITLE
Add a simpler example to the doc.

### PR DESCRIPTION
**Pull request by JulienPalard**
I think that a first short example make a smoother introduction, at least I think that's what I searched while opening the doc, so I added it.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -32,6 +32,28 @@ Using MediaInfo
 There isn't much to this library so instead of a lot of documentation it is
 probably best to just demonstrate how it works:
 
+Get informations on an image
```
Comment (sbraz) [inline docs/index.rst:35]: I'm pretty sure information isn't countable here.

File: docs/index.rst [hunk 2]
```diff
@@ -32,6 +32,28 @@ Using MediaInfo
...
+----------------------------
+
+.. code-block:: python
+
+  from pymediainfo import MediaInfo
+
+  media_info = MediaInfo.parse("/home/mdk/julien.jpg")
```
Comment (sbraz) [inline docs/index.rst:42]: Can we change that to a more generic location? Something like `"image.jpg"`?

File: docs/index.rst [hunk 3]
```diff
@@ -32,6 +32,28 @@ Using MediaInfo
...
+
+  for track in media_info.tracks:
+      if track.track_type == "Image":
+          print(f"{track.format} of {track.width}×{track.height} pixels.")
+
+This print ssomething like:
```
Comment (sbraz) [inline docs/index.rst:48]: Can you please fix the typo?

File: docs/index.rst (hunk 2 above)
Comment (JulienPalard) [reply docs/index.rst:42]: yes :)

File: docs/index.rst (hunk 1 above)
Comment (JulienPalard) [reply docs/index.rst:35]: fixed.

**Comment by JulienPalard** [comment]
> Do you see anything else that would be worth adding?

I'll PR it if I see it. My idea was to add to the doc the specific topic I was searching for in the doc.

> Maybe mentioning that objects can be dumped with `to_data()`?

Maybe. There's already an example of it, in the pprint, but that's ... short.

> I've changed the PR title to fix a typo, can you do the same with the commit message?

Done, I bet it's not the first time I do this one.

> Is it OK if I make a few changes to your commit while merging it? I might reword a thing or two.

Yes, totally OK.

**Comment by sbraz** [comment]
Hi @JulienPalard,
What do you think of [my expanded version of this PR](https://github.com/sbraz/pymediainfo/compare/pr_improve_doc?expand=1)?

**Comment by JulienPalard** [comment]
https://github.com/sbraz/pymediainfo/commit/d46d63d0e4a3194d3a3a6e635cd05a2199d2d4b1 Looks good to me :)

**Comment by sbraz** [comment]
Merci, merged :)

**Pull request closed without merging**